tests/test_markup.py

```diff
@@ -98,9 +51,296 @@ def test_relative_path_conversion(tmp_path):
 
 def test_absolute_path_conversion(tmp_path):
     abs_paths = [
-        '[""file:///abs"".jpg]', '[""%s/aha 1"".jpg]' % tmp_path,
-        '[abs.pdf ""file:///abs.pdf""]', '[abs.pdf ""%s/abs.pdf""]' % tmp_path,
+        '[""file:///abs"".jpg]', '[""{}/aha 1"".jpg]'.format(tmp_path),
+        '[abs.pdf ""file:///abs.pdf""]', '[abs.pdf ""{}/abs.pdf""]'.format(tmp_path),
         'www.google.com', 'www.google.com/page.php']
 
     for path in abs_paths:
         assert path == _convert_paths(path, tmp_path)
+
+
+def test_html_export_contains_day_fragment_reference_element(tmp_path):
+    from rednotebook.data import Day, Month
+    import datetime
+    date = datetime.date(2019, 10, 21)
```

Comment:
I know this is old code, but can't we make the imports at the top of the file?

Reply:
Sure :)